gemini/use-cases/retrieval-augmented-generation/intra_knowledge_qna.ipynb

```diff
@@ -0,0 +1,832 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.    class CustomVertexAIEmbeddings(VertexAIEmbeddings):
</pre><p>Is there a reason to create a custom class instead of using the existing Embeddings class from LangChain? Or you can edit the existing class in the langchain library?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/483/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Removed the custom class. Directly created VertexAIEmbeddings object and passed batch_size parameter.</p>